assets/component-mega-menu.css

```diff
@@ -0,0 +1,67 @@
+.mega-menu {
+  position: static;
+}
+
+.mega-menu__content {
+  background-color: rgb(var(--color-background));
+  border-left: 0;
+  border-radius: 0;
+  border-right: 0;  
+  left: 0;
+  max-height: calc(var(--viewport-height, 100vh) - var(--header-bottom-position, 20rem) - 4rem);
+  overflow-y: auto;
+  padding-bottom: 2.4rem;
+  padding-top: 2.4rem;
+  position: absolute;
+  right: 0;
+  top: 100%;
+  z-index: -1;
+}
+
+.header-wrapper--border-bottom .mega-menu__content {
+  border-top: 0;
+}
+
+.js .mega-menu__content {
+  opacity: 0;
+  transform: translateY(-1.5rem);
+}
+
+.mega-menu[open] .mega-menu__content {
+  animation: animateMenuOpen var(--duration-default) ease;
+  animation-fill-mode: forwards;
+}
+
+@media (prefers-reduced-motion) {
+  .mega-menu[open] .mega-menu__content {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+.mega-menu__list {
+  display: grid;
+  gap: 2.4rem 4rem;
+  grid-template-columns: repeat(6, minmax(0, 1fr));
+  list-style: none;
+}
+
+.mega-menu__link {
+  color: rgba(var(--color-foreground), 0.75);
+  display: block;
+  line-height: 1.4;
+  padding-bottom: 0.8rem;
+  padding-top: 0.8rem;
+  text-decoration: none;
+  transition: text-decoration var(--duration-short) ease;
+}
+
+.mega-menu__link:hover,
+.mega-menu__link--active {
+  color: rgb(var(--color-foreground));
+  text-decoration: underline;
+}
+
+.mega-menu__link--active:hover {
+  text-decoration-thickness: 0.2rem;
+}
```

Comment:
Here as well, we apply a similar rule on: 
```css
list-menu__item--active:hover {
  text-decoration-thickness: 0.2rem;
}
```

Which maybe could be a common styling 🤔

Reply:
I agree this could benefit from some refactoring. The styles for menu links (dropdown + mega menu styles) are split between:

- `.header__menu-item`
- `.header__menu-item:hover`
- `.header__menu-item span`
- `.header__menu-item:hover span`
- `.list-menu__item`
- `.list-menu__item--active`
- `.list-menu__item--active:hover`
- `.mega-menu__link`
- `.mega-menu__link:hover`
- `.mega-menu__link--active`
- `.mega-menu__link--active:hover`
- `.link`

I'd like to consolidate this, but would prefer to do this in a follow-up PR at this point, because we'd have to make some changes in the markup of both menu styles and would have to re-test all use cases to ensure there are no regressions.